### ISSUE TITLE
[website add-on] header navigation add-on a11y bugs

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-website#711]
copied from https://github.com/IBM-Design/design-website/pull/36 . cc: @dakahn 

> Hi found a few accessibility issues testing on the latest Firefox and Chrome:

_- [ ] 1)At larger breakpoints keyboard navigation gets trapped on Approach. I can't activate the drop down with either enter or space bar and I can't tab to the next item. Inspecting the code shows that the remaining 3 navbar menu items have a taborder of -1
![screen shot 2019-01-14 at 2 23 47 pm](https://user-images.githubusercontent.com/40970507/51139619-ff639780-1809-11e9-9785-cb8f302bf215.png)_

- [x] ~2)On either breakpoint Skip to main content can't be activated with either enter or space.~

- [ ] 3) On smaller breakpoints when the hamburger menu is first opened approach should receive focus since it's the first item in the menu.

- [ ] 4)The hamburger menu doesn't announce it has been opened to the user.

_- [ ] 5)On smaller breakpoints when the hamburger drop down is activated users should be able to use the arrow keys to navigate menu items and tab should be preserved for moving out of the menu. 
[WAI-ARIA Authoring Practices for a menu/menuitems keyboard interactivity](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton). 
[Here's an example of the keyboard navigation pattern I reference all the time](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html)_
